text/0000-export-to-oci.md

```diff
@@ -0,0 +1,213 @@
+
+```=shell
+> export CNB_LAYOUT_DIR=oci
+>  /cnb/lifecycle/exporter -daemon my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+
+### Exporting using the command line flag
+
+```=shell
+>  /cnb/lifecycle/exporter -daemon -layout oci my-app-image
+> tree /oci
+.
+└── oci/
+    └── my-app-image/
+        ├── blobs/
+        │   └── sha256/
+        │       └── 01..
+        ├── index.json
+        └── oci-layout
+
+```
+As we can see there is a new folder called `oci` and inside that folder we can find our application image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format
+
+Attempts to use this feature when the `-daemon` flag is not used could be ignore or show some warnings messages.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+The lifecycle phases affected by this new behavior is: [Export](https://buildpacks.io/docs/concepts/components/lifecycle/export/)
+
+At high level view of the propose solution can be summarize with the following container diagram from the C4 model
+
+![](https://i.imgur.com/XhtvT9j.png)
+
+Notice that we are relying on the OCI format Specification to expose the data for `Platforms`
+
+The following new input is proposed to be added to this phase
+
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<layout>`      |  `CNB_LAYOUT_DIR` | "" | The root directory where the OCI image will be written. The presence of a none empty value for this environment variable will enable the feature. |
+
+
+- When the exporter is executed WITH the flag `daemon` it will check for the presence of the flag `layout` or the environment variable `CNB_LAYOUT_DIR`
+- IF any of the values are present THEN
+  - It will create a folder name `<image>` located at the path defined `<layout>` or `CNB_LAYOUT_DIR`
+  - In parallel of exporting the Image to the Daemon it will save the final Image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format in the directory created before. The image will be saved using **uncompressed** layers
+  - It will calculate the digest of the manifest of the compressed layers and write that value into the report.toml file
+  - It will update the report.toml file with all the tags and require information to verify the image once it is pushed into a registry
+- OTHERWISE it will behave as it is right now
+
+#### `report.toml` (TOML)
+
+The new information to be  added into the `report.toml` file can be summarize as follows:
+
+```toml
+[export]
+[[export.oci]]
+digest = "<image digest>"
+manifest-size = "<manifest size in bytes>"
+compression-algorithm = "<used by gcr>"
+library-url = "https://github.com/google/go-containerregistry/"
+library-language="go"
```

Comment:
If we do https://github.com/buildpacks/rfcs/pull/203/files#r819905382 this would go away.

Reply:
Yes, I will remove this section